src/content/apis/travel/IRCTC_API.md

```diff
@@ -0,0 +1,10 @@
+---
+name: IRCTC API
+description: An API with various functionalities for Indian railways IRCTC API. We are not affiliated with Indian railways IRCTC in any form, and this is not an official API, but all the results are real-time and accurate.
```

Comment:
Can we please shorten the description a bit?

Reply:
Ok I'll do it